src/constants.js

```diff
@@ -0,0 +1,14 @@
+export const VERSION = '0.0.10'
+
+export const digidoc_mime = 'application/x-digidoc'
+export const digidoc_chrome = 'TokenSigning'
+
+export const USER_CANCEL = 'user_cancel'
+export const NO_CERTIFICATES = 'no_certificates'
+export const INVALID_ARGUMENT = 'invalid_argument'
+export const TECHNICAL_ERROR = 'technical_error'
+export const NO_IMPLEMENTATION = 'no_implementation'
+export const NOT_ALLOWED = 'not_allowed'
+
+export const localServiceURL = 'https://local.ria.ee'
```

Comment:
Actually it is a bigger topic I would like to discuss.
Has local servlet as a pluginless backend been discussed? What are the reasons that contradict to this option? 

So actually it currently should stay in a separate branch.